components/server/src/ome/logic/AWTScaleService.java

```diff
@@ -58,6 +56,7 @@ public BufferedImage scaleBufferedImage(BufferedImage image, float xScale,
         // graphics2D.setRenderingHint(RenderingHints.KEY_INTERPOLATION,
         // RenderingHints.VALUE_INTERPOLATION_BILINEAR);
         graphics2D.drawImage(image, 0, 0, thumbWidth, thumbHeight, null);
+        graphics2D.dispose();
```

Comment:
Worth `try/finally`s?

Reply:
I thought probably not given that they do get cleaned up anyway, it's not like leaking file descriptors, and it's pretty unlikely that `drawImage` and its like are going to throw, so just trying to balance code clutter with likely benefit.